public/static/locales/en/vice.json

```diff
@@ -0,0 +1,9 @@
+{
+    "accessRequestTitle": "Request access to VICE analyses (e.g: Jupyter, RStudio etc...)",
```

Comment:
Picking a random spot - for these messages maybe we should pick a particular word to use.  Here we're saying `VICE analyses`, in another message it says `... not permitted to launch VICE apps`, and in one other it says `... currently running VICE jobs`.  We've had some discussions in Slack about how some users are confused by "analyses" in regards to VICE, so it's stuck in my head as something to think about 😄

Reply:
Good catch. You are right, we should be consistent in our messaging.